cmd/access/access_node_builder.go

```diff
@@ -262,3 +275,10 @@ func (builder *FlowAccessNodeBuilder) initNetwork(nodeID module.Local,
 
 	return net, nil
 }
+
+func unstakedNetworkMsgValidators(selfID flow.Identifier) []network.MessageValidator {
+	return []network.MessageValidator{
+		// filter out messages sent by this node itself
+		validator.NewSenderValidator(selfID),
```

Comment:
We should filter out all *except* those from the upstream staked node, right?

Reply:
yes i will be creating a new filter for that in a separate PR (and also rename this filter to `LoopbackValidator` or something)

Reply:
https://github.com/onflow/flow-go/pull/1100